test/e2e/slack_test.go

```diff
@@ -312,8 +311,6 @@ func TestSlack(t *testing.T) {
 					return strings.Contains(msg.Text, heredoc.Doc(fmt.Sprintf("Cluster: %s", appCfg.ClusterName))) &&
 						strings.Contains(msg.Text, "local-path-provisioner") &&
 						strings.Contains(msg.Text, "coredns") &&
-						strings.Contains(msg.Text, "traefik") &&
```

Comment:
I also mentioned this part on slack, https://kubeshop.slack.com/archives/C03MRCX7UE9/p1660819668105749